tests/fixtures/git_repo.py

```diff
@@ -926,11 +926,11 @@ def simulate_change_commits_n_rtn_changelog_entry(
     file_in_repo: str,
 ) -> SimulateChangeCommitsNReturnChangelogEntryFn:
     def _simulate_change_commits_n_rtn_changelog_entry(
-        git_repo: Repo, commit_msgs: Sequence[CommitDef]
+        git_repo: Repo, commit_msgs: Sequence[CommitDef], file_path: str | None
```

Comment:
This is the change described in the PR title. If I create commits in the feature branch before merging the default branch, it leads to merge conflicts. Making changes in a different file avoids this issue.